### PR TITLE
Ory test on evaluation

### PR DESCRIPTION
**Pull request by dariusztutaj**
See: #638

### CODE REVIEW

File: pkg/reconciler/instances/ory/test/ory_integration_test.go [hunk 1]
```diff
@@ -77,12 +77,75 @@ func TestOryIntegrationProduction(t *testing.T) {
 		setup.ensureSecretIsDeployed(t, credsName)
 	})
 }
+func TestOryIntegrationEvaluation(t *testing.T) {
+	skipTestIfDisabled(t)
 
-func (s *oryTest) getHorizontalPodAutoscaler(t *testing.T, name string) *autoscalingv1.HorizontalPodAutoscaler {
```
Comment (cnvergence) [inline pkg/reconciler/instances/ory/test/ory_integration_test.go:81]: I'm generally okay with simplifying this function and moving error checking up.
Could you please also refactor the prod test?

File: pkg/reconciler/instances/ory/test/ory_integration_test.go [hunk 2]
```diff
@@ -77,12 +77,75 @@ func TestOryIntegrationProduction(t *testing.T) {
...
+		options := metav1.ListOptions{
+			LabelSelector: "app.kubernetes.io/name=hydra-maester",
+		}
+		podsList, err := setup.getPods(options)
+		require.NoError(t, err)
+		require.Equal(t, 1, len(podsList.Items))
+		setup.logger.Infof("Single pod %v is deployed for app: hydra-maester", podsList.Items[0].Name)
+	})
+	/*
+		t.Run("ensure that ory secrets are deployed", func(t *testing.T) {
```
Comment (cnvergence) [inline pkg/reconciler/instances/ory/test/ory_integration_test.go:132]: We can move it to the general Ory test, WDYT?

File: pkg/reconciler/instances/ory/test/helpers.go [hunk 3]
```diff
@@ -61,3 +61,8 @@ func isProductionProfile() bool {
 
 	return ok && prodEnabled == "production"
 }
+func isEvaluationProfile() bool {
```
Comment (werdes72) [inline pkg/reconciler/instances/ory/test/helpers.go:64]: Missing newline between functions

File: pkg/reconciler/instances/ory/test/ory_integration_test.go [hunk 4]
```diff
@@ -77,12 +77,75 @@ func TestOryIntegrationProduction(t *testing.T) {
...
+	t.Run("ensure that ory-hydra-maester pod is deployed", func(t *testing.T) {
+		options := metav1.ListOptions{
+			LabelSelector: "app.kubernetes.io/name=hydra-maester",
+		}
+		podsList, err := setup.getPods(options)
+		require.NoError(t, err)
+		require.Equal(t, 1, len(podsList.Items))
+		setup.logger.Infof("Single pod %v is deployed for app: hydra-maester", podsList.Items[0].Name)
+	})
+	/*
```
Comment (mjakobczyk) [inline pkg/reconciler/instances/ory/test/ory_integration_test.go:131]: Please remove unnecessary comments

File: pkg/reconciler/instances/ory/test/ory_integration_test.go [hunk 5]
```diff
@@ -77,12 +77,75 @@ func TestOryIntegrationProduction(t *testing.T) {
...
+
+	t.Run("ensure that ory-oathkeeper pod is deployed", func(t *testing.T) {
+		options := metav1.ListOptions{
+			LabelSelector: "app.kubernetes.io/name=oathkeeper",
+		}
+		podsList, err := setup.getPods(options)
+		require.NoError(t, err)
+		require.Equal(t, 1, len(podsList.Items))
+		setup.logger.Infof("Single pod %v is deployed for app: oathkeeper", podsList.Items[0].Name)
+	})
```
Comment (mjakobczyk) [inline pkg/reconciler/instances/ory/test/ory_integration_test.go:112]: Please add new line between test cases

File: pkg/reconciler/instances/ory/test/ory_integration_test.go [hunk 6]
```diff
@@ -77,12 +77,75 @@ func TestOryIntegrationProduction(t *testing.T) {
...
+	t.Run("ensure that ory-hydra pod is deployed", func(t *testing.T) {
+		options := metav1.ListOptions{
+			LabelSelector: "app.kubernetes.io/name=hydra",
+		}
+		podsList, err := setup.getPods(options)
+		require.NoError(t, err)
+		require.Equal(t, 1, len(podsList.Items))
+		setup.logger.Infof("Single pod %v is deployed for app: hydra", podsList.Items[0].Name)
+	})
```
Comment (mjakobczyk) [inline pkg/reconciler/instances/ory/test/ory_integration_test.go:121]: Please add new line between test cases

**Review by mjakobczyk — DISMISSED** [review]

File: pkg/reconciler/instances/ory/test/ory_integration_test.go (hunk 2 above)
Comment (dariusztutaj) [reply pkg/reconciler/instances/ory/test/ory_integration_test.go]: moved

**Review by cnvergence — APPROVED** [review]
LGTM

**Pull request merged**